qutip/solver/ode/evolver.py

```diff
@@ -0,0 +1,141 @@
+
+from ..evolver import Evolver, evolver_collection
+from .verner7efficient import vern7
+from .verner9efficient import vern9
+from .wrapper import QtOdeFuncWrapper
+import numpy as np
+from ...core import data as _data
```

Comment:
Same comment about relative vs absolute imports.

Reply:
The last line should _probably_ be `from qutip import data as _data` since it's a separate package and exported to the public interface, but I'm fairly sure I've also been a bit lax elsewhere with that one too. The others are within the same package, and it's easy to see that at a glance.